server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -30,7 +30,9 @@ import (
 )
 
 var (
-	detailedStatsEnabled = flag.Bool("cache.detailed_stats_enabled", false, "Whether to enable detailed stats recording for all cache requests.")
+	detailedStatsEnabled       = flag.Bool("cache.detailed_stats_enabled", false, "Whether to enable detailed stats recording for all cache requests.")
+	scorecardResultsExpiration = flag.Duration("cache.detailed_stats_expiration", 3*time.Hour, "How long to go without receiving any cache requests for an invocation before deleting the invocation's detailed results from the metrics collector. Has no effect if cache.detailed_stats_enabled is not set.")
+	countExpiration            = flag.Duration("cache.count_expiration", 24*time.Hour, "How long to go without receiving any cache requests for an invocation before deleting the invocation's counts from the metrics collector.")
```

Comment:
nit: maybe `s/_expiration/_ttl/` since it seems we use _ttl for other flags

Reply:
sgtm!